api/authenticator.py

```diff
@@ -158,7 +157,20 @@ def __repr__(self):
             self.permanent_id, self.authorization_identifier,
             self.username
         )
-        
+
+    @hybrid_property
+    def fines(self):
+        return self._fines
+
+    @fines.setter
+    def _set_fines(self, value):
+        """When setting patron fines, only store the numeric portion of 
+        a Money object.
+        """
+        if isinstance(value, Money):
+            value = value.amount
+        self._fines = value
```

Comment:
I haven't seen an example of this and it seems unlikely. Even if it was true, this code would work so long as the maximum allowable fine was also counted in pennies.